asmcomp/amd64/emit.mlp

```diff
@@ -70,6 +70,9 @@ let stack_offset = ref 0
 
 (* Layout of the stack frame *)
 
+(* Must be consistent with byterun/caml/config.h:Stack_threshold *)
+let stack_threshold_size = 128 (* bytes *)
```

Comment:
This is already available as Config.stack_threshold (which is measured in words).